src/runners/wteg-feedback-runner.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WTEGFeedbackWidget from '../../components/feedback/wteg_feedback_widget.react';
+
+{
+	const detailsPage = document.querySelector('.details-page'),
+		path = window.location.pathname,
+		url = window.location.href,
+		wtegWrapperId = 'what-to-expect',
```

Comment:
Please make sure it's changed to `what-to-expect-form`

Reply:
Done